react-components/my-app/src/Image.tsx

```diff
@@ -0,0 +1,8 @@
+export function Image() {
+  return (
+    <img
+      src={'./public/tumblr_.jpg'}
```

Comment:
This works because you are returning a string from within the `{}`. However, for string values, it's simplest and cleanest to just use `"`. That is, `<img src="/tumbler_.jpg" />`.

Note, also, that in Vite you do not need to use `./public`. Instead you just use `/` and Vite automatically looks in the public folder for the file.